fastlane/Fastfile

```diff
@@ -866,49 +865,85 @@ platform :ios do
   end
 
   desc "Build and deploy PurchaseTesterSwiftUI"
-  lane :deploy_purchase_tester do
-    latest_changelog = File.read("#{File.dirname(__FILE__)}/../#{changelog_latest_path}")
```

Comment:
quite a few changes here, but they ultimately boil down to updating things so that we're running the lane from the repo root rather than PurchaseTester root. 
This is because we do need to get the `RevenueCat` SDK that's generated at the top level. It wasn't needed before since we were using an SPM dependency on a hardcoded commit instead, but the local dependency is much easier to work with on a daily basis.